paper/paper.md

```diff
@@ -0,0 +1,141 @@
+    - name: Emily Giles
+      affiliation: 4
+    - name: Katlyn Greene
+      affiliation: 4
+    - name: Sung Cho
+      affiliation: 4
+    - name: Shak Ragoler
+      affiliation: 4
+    - name: Nicholas Weber
+      orcid: 0000-0002-6008-3763
+      affiliation: 1
+
+affiliations:
+    - name: University of Washington iSchool, University of Washington, Seattle
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: Washington University, St. Louis
+      index: 3
+    - name: Independent Contributor
+      index: 4
+
+date: 29 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Cities, counties, and states throughout the USA are bound by law to archive recordings of public meetings. Most local governments comply with these laws by posting documents, audio, or video recording online. Council Data Project (CDP) is three open-source tools that improve the accessibility of local government data by systematically collecting, transforming, and re-publishing this data to the web. The software provided here can be used by civic technologists to promote government transparency, or using the processed data in support of research in public policy, natural language processing and machine learning, or information retrieval and discovery.
+
+# Statement of Need
+
+Comparative research into municipal governance in the USA is often prohibitively difficult due to a broad federal system where states, counties, and cities divide legislative powers differently. This has contributed to the lack of large-scale quantitative studies of municipal government, and impeded necessary research into effective procedural elements of administrative and legislative processes [@Trounstine:2009]. Council Data Project (CDP) enables large-scale quantitative studies by generating standardized municipal governance corpora - including legislative voting records, timestamped transcripts, and full legislative matter attachments (related reports, presentations, amendments, etc.).
+
+## Related Work
+
+Work in extracting and repackaging government data into machine-readable and experiment ready datasets has historically happened in fields with highly structured data, such as meteorology [@Sparks2017] and legal review and monitoring [@courtlistener]. Notably, there has been prior work in extracting and repackaging municipal government data with [Councilmatic](https://github.com/codeforamerica/councilmatic) [@councilmatic]. However, this work largely aimed to make municipal data more accessible to a general public, and didn't add any specific data processing to expand the research capabilities of the produced dataset. Recent advances in natural language processing have made it possible to conduct large-scale transcript-based studies on the effects of gender, ideology, and seniority in Supreme Court oral argument [@jacobi2017] and the effects that information communication technology has on civic participation [@einstein2021].
```

Comment:
Most meteorology data comes from central gov offices yes. Ever heard the term "Met Office" https://www.metoffice.gov.uk/

I watched a great talk once how one of the oldest datasets that humans have always cared about is "what will the weather be like tomorrow or this week". It matters for so much. So largely meteorological data is handled by central offices because of how crucial it is.

Fun fact, the met office in the UK is one of the best "open source" orgs I know of. They are both constantly pushing the envelope of best practices for data publishing and access. And because there datasets are massive they must create new tools.

Reply:
Lol well, yep those are indeed my fun facts of the day!